### PR TITLE
Menu cleanup

### PR DESCRIPTION
**Pull request by dannon**
@assuntad23 Feel free to take this branch over ;)

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [ ] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these contributions under [Galaxy's current license](https://github.com/galaxyproject/galaxy/blob/dev/LICENSE.txt).
- [x] I agree to allow the Galaxy committers to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT). If this condition is an issue, uncheck and just let us know why with an e-mail to galaxy-committers@lists.galaxyproject.org.

### CODE REVIEW

File: doc/source/releases/21.09_announce.rst [hunk 1]
```diff
@@ -125,7 +125,6 @@ config/galaxy.yml.sample:galaxy
 -  job_runner_monitor_sleep
 -  mailing_join_body
 -  mailing_join_subject
--  mailing_lists_url
```
Comment (nsoranzo) [inline doc/source/releases/21.09_announce.rst]: Please revert, no reason to change old release notes unless they are factually wrong.

File: lib/galaxy/config/schemas/config_schema.yml [hunk 2]
```diff
@@ -1473,7 +1474,7 @@ mapping:
 
       citation_url:
         type: str
-        default: https://galaxyproject.org/citing-galaxy
+        default: https://academic.oup.com/nar/article/48/W1/W395/5849904
```
Comment (nsoranzo) [inline lib/galaxy/config/schemas/config_schema.yml]: I don't understand the reason for this change, the "Citing Galaxy" page have clear directions on which paper(s) to cite.

File: lib/tool_shed/webapp/templates/webapps/tool_shed/base_panels.mako [hunk 3]
```diff
@@ -138,17 +138,12 @@
...
                         [_('Tool Shed Wiki'), app.config.get( "wiki_url", "https://galaxyproject.org/toolshed" ), "_blank" ],
                         [_('Support'), app.config.get( "support_url", "https://galaxyproject.org/support" ), "_blank" ],
-                        [_('Search'), app.config.get( "search_url", "http://galaxyproject.org/search/" ), "_blank" ],
-                        [_('Mailing Lists'), app.config.get( "mailing_lists_url", "https://galaxyproject.org/mailing-lists" ), "_blank" ],
-                        [_('Videos'), app.config.get( "screencasts_url", "https://vimeo.com/galaxyproject" ), "_blank" ],
+                        [_('Videos'), app.config.get( "screencasts_url", "https://www.youtube.com/c/galaxyproject" ), "_blank" ],
                         [_('Wiki'), app.config.get( "wiki_url", "http://galaxyproject.org/" ), "_blank" ],
-                        [_('How to Cite Galaxy'), app.config.get( "citation_url", "https://galaxyproject.org/citing-galaxy" ), "_blank" ]
+                        [_('How to Cite Galaxy'), app.config.get( "citation_url", "https://academic.oup.com/nar/article/48/W1/W395/5849904" ), "_blank" ]
```
Comment (nsoranzo) [inline lib/tool_shed/webapp/templates/webapps/tool_shed/base_panels.mako]: This should probably be "How to cite the Tool Shed" ?

File: test/unit/config/config_manage/1607_root_filters/config/galaxy.ini [hunk 4]
```diff
@@ -524,23 +524,17 @@ nglims_config_file = tool-data/nglims.yaml
 # The URL linked by the "Galaxy/brand" text.
 #logo_url = /
 
-# The URL linked by the "Wiki" link in the "Help" menu.
+# The URL linked by the "Community Hub" link in the "Help" menu.
```
Comment (nsoranzo) [inline test/unit/config/config_manage/1607_root_filters/config/galaxy.ini]: Please revert all changes to this file, this is not meant to be modified.

File: test/unit/config/config_manage/1607_root_samples/config/galaxy.ini [hunk 5]
```diff
@@ -524,23 +524,17 @@ nglims_config_file = tool-data/nglims.yaml
 # The URL linked by the "Galaxy/brand" text.
 #logo_url = /
 
-# The URL linked by the "Wiki" link in the "Help" menu.
+# The URL linked by the "Community Hub" link in the "Help" menu.
```
Comment (nsoranzo) [inline test/unit/config/config_manage/1607_root_samples/config/galaxy.ini]: As above, this file is not meant to be modified.

File: lib/tool_shed/webapp/templates/webapps/tool_shed/base_panels.mako [hunk 6]
```diff
@@ -138,17 +138,12 @@
                 ## Help tab.
                 <%
                     menu_options = []
-                    qa_url = app.config.get( "qa_url", None )
-                    if qa_url:
-                        menu_options = [ [_('Galaxy Q&A'), qa_url, "_blank" ] ]
                     menu_options.extend( [
                         [_('Tool Shed Wiki'), app.config.get( "wiki_url", "https://galaxyproject.org/toolshed" ), "_blank" ],
```
Comment (nsoranzo) [inline lib/tool_shed/webapp/templates/webapps/tool_shed/base_panels.mako]: Rename `Tool Shed Wiki` as well? Not sure how though!

File: lib/tool_shed/webapp/templates/webapps/tool_shed/base_panels.mako [hunk 7]
```diff
@@ -138,17 +138,12 @@
...
                         [_('Support'), app.config.get( "support_url", "https://galaxyproject.org/support" ), "_blank" ],
-                        [_('Search'), app.config.get( "search_url", "http://galaxyproject.org/search/" ), "_blank" ],
-                        [_('Mailing Lists'), app.config.get( "mailing_lists_url", "https://galaxyproject.org/mailing-lists" ), "_blank" ],
-                        [_('Videos'), app.config.get( "screencasts_url", "https://vimeo.com/galaxyproject" ), "_blank" ],
+                        [_('Videos'), app.config.get( "screencasts_url", "https://www.youtube.com/c/galaxyproject" ), "_blank" ],
                         [_('Wiki'), app.config.get( "wiki_url", "http://galaxyproject.org/" ), "_blank" ],
```
Comment (nsoranzo) [inline lib/tool_shed/webapp/templates/webapps/tool_shed/base_panels.mako]:
```suggestion
                        [_('Galaxy Community Hub'), app.config.get( "wiki_url", "http://galaxyproject.org/" ), "_blank" ],
```

Or drop it?

File: lib/galaxy/config/schemas/config_schema.yml (hunk 2 above)
Comment (assuntad23) [reply lib/galaxy/config/schemas/config_schema.yml]: This is what @nekrut requested. Any insight, Anton?

File: lib/tool_shed/webapp/templates/webapps/tool_shed/base_panels.mako (hunk 3 above)
Comment (assuntad23) [reply lib/tool_shed/webapp/templates/webapps/tool_shed/base_panels.mako]: Again, I was just trying to find any references to the things @nekrut wanted to change and change them. If it should be something else, then I'm happy to oblige.

File: doc/source/releases/_thanks.rst [hunk 8]
```diff
@@ -1,4 +1,4 @@
-To stay up to date with Galaxy's progress watch our `screencasts <https://vimeo.com/galaxyproject>`__,
+To stay up to date with Galaxy's progress watch our `screencasts <https://www.youtube.com/c/galaxyproject>`__,
```
Comment (assuntad23) [inline doc/source/releases/_thanks.rst:1]: @nsoranzo This .rst is okay?

File: lib/tool_shed/webapp/templates/webapps/tool_shed/base_panels.mako (hunk 6 above)
Comment (assuntad23) [reply lib/tool_shed/webapp/templates/webapps/tool_shed/base_panels.mako]: What do you think it should be renamed to? I don't even see this as a option in the Help Menu, in fact.

File: doc/source/releases/_thanks.rst (hunk 8 above)
Comment (nsoranzo) [reply doc/source/releases/_thanks.rst:1]: It is! It's included in all release note pages, so it makes sense to update.

File: lib/tool_shed/webapp/templates/webapps/tool_shed/base_panels.mako (hunk 6 above)
Comment (nsoranzo) [reply lib/tool_shed/webapp/templates/webapps/tool_shed/base_panels.mako]: This is part of the [ToolShed](https://toolshed.g2.bx.psu.edu/)'s Help menu, not Galaxy. Maybe we can call this "About" ?

File: lib/tool_shed/webapp/templates/webapps/tool_shed/base_panels.mako (hunk 3 above)
Comment (nsoranzo) [reply lib/tool_shed/webapp/templates/webapps/tool_shed/base_panels.mako]: As mentioned above, this is part of the [Tool Shed](https://toolshed.g2.bx.psu.edu/)'s Help menu, so I think it makes more sense to mention the Tool Shed and link to either https://doi.org/10.1186/gb4161 or the "Citing Galaxy" Hub page (as before).

**Comment by assuntad23** [comment]
This PR ties directly to #13320 and #13321, as I've removed both of those options from the menu in this PR.  

@nekrut Can you take a look at this? @nsoranzo had a question for you regarding the "How to Cite" link update.

File: lib/galaxy/config/schemas/config_schema.yml (hunk 2 above)
Comment (nekrut) [reply lib/galaxy/config/schemas/config_schema.yml]: We are trying to decrease the number of hub pages one needs to maintain. An alternative could be an API call to PubMed with PMIDs of publications we want

Comment (bgruening) [reply lib/galaxy/config/schemas/config_schema.yml]: I agree with lowering the hub pages to maintain. In this case, however, we need to maintain either the hub page or the Galaxy source. I prefer the hub page here, this gives us more freedom.

Comment (nekrut) [reply lib/galaxy/config/schemas/config_schema.yml]: @assuntad23 -> keep the hub page

**Comment by assuntad23** [comment]
@hexylena @nekrut @nsoranzo @bgruening I reverted the citation link to the one on the hub. 
Anything else to change? Or does this look good to all parties now? 

Thanks!

**Comment by nsoranzo** [comment]
@assuntad23 Thanks a lot for following up on this! Only https://github.com/galaxyproject/galaxy/pull/13288/files#r801197206 is left from my side (probably dropping the Wiki link altogether being my preferred option).

**Comment by assuntad23** [comment]
> @assuntad23 Thanks a lot for following up on this! Only https://github.com/galaxyproject/galaxy/pull/13288/files#r801197206 is left from my side (probably dropping the Wiki link altogether being my preferred option).

Just to confirm, you want this line `  [_('Wiki'), app.config.get( "wiki_url", "http://galaxyproject.org/" ), "_blank" ],` dropped?

**Comment by assuntad23** [comment]
> @assuntad23 Thanks a lot for following up on this! Only https://github.com/galaxyproject/galaxy/pull/13288/files#r801197206 is left from my side (probably dropping the Wiki link altogether being my preferred option).

Hopefully resolved now!

File: client/src/nls/es/locale.js [hunk 9]
```diff
@@ -40,12 +40,10 @@ define({
 
     Videos: "Videos",
 
-    Wiki: "Wiki",
+    "Community Hub": "Centro Comunitario",
 
     "How to Cite Galaxy": "Cómo citar Galaxy",
 
-    "Interactive Tours": "Tours Interactivos",
```
Comment (nsoranzo) [inline client/src/nls/es/locale.js]: Hopefully tours can come back, maybe we can keep the translations?

File: lib/tool_shed/webapp/templates/webapps/tool_shed/base_panels.mako [hunk 10]
```diff
@@ -138,17 +138,13 @@
...
                     menu_options = []
-                    qa_url = app.config.get( "qa_url", None )
-                    if qa_url:
-                        menu_options = [ [_('Galaxy Q&A'), qa_url, "_blank" ] ]
                     menu_options.extend( [
-                        [_('Tool Shed Wiki'), app.config.get( "wiki_url", "https://galaxyproject.org/toolshed" ), "_blank" ],
+                        [_('About Tool Shed'), app.config.get( "wiki_url", "https://galaxyproject.org/toolshed" ), "_blank" ],
                         [_('Support'), app.config.get( "support_url", "https://galaxyproject.org/support" ), "_blank" ],
                         [_('Search'), app.config.get( "search_url", "http://galaxyproject.org/search/" ), "_blank" ],
                         [_('Mailing Lists'), app.config.get( "mailing_lists_url", "https://galaxyproject.org/mailing-lists" ), "_blank" ],
```
Comment (nsoranzo) [inline lib/tool_shed/webapp/templates/webapps/tool_shed/base_panels.mako]:
```suggestion
```

**Comment by dannon** [comment]
@assuntad23 Thanks for sorting all this out :)

**Pull request merged**